version.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"fmt"
+	"runtime"
+)
+
+// populated by -ldflags from Makefile
+var version = "unknown"
```

Comment:
You could have a runtime check to make sure that the version is populated via make:
https://github.com/mattbostock/leavediary/blob/adc0179a76e0fe6281506041baf103bbb373b052/main.go#L86-L88